.github/workflows/validate.yaml

```diff
@@ -0,0 +1,112 @@
+name: VALIDATE
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * SUN'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        BUILD_TYPE: [Release, Debug]
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Dependencies
+        run: sudo apt-get update && sudo apt-get install -yq openmpi-bin libfftw3-dev python3-dev libboost-all-dev python3-h5py
+
+      - name: Configure
+        run: cmake -S ${{github.workspace}} -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{ matrix.BUILD_TYPE }} -DEXTERNAL_MPI4PY=OFF
+
+      - name: Build
+        run: cmake --build ${{github.workspace}}/build --target all -j 2
+
+      #    - name: Install
+      #      run: cmake --install ${{github.workspace}}/build
+
+      - name: Test
+        working-directory: ${{github.workspace}}/build
+        run: ctest --rerun-failed --output-on-failure -j 2
+
+  documentation:
+    runs-on: ubuntu-latest
+    environment: CI
+    steps:
+      - name: install dependencies
+        # very ugly but did not get line breaks working...
```

Comment:
I think you can do `run: |` here and then use `\` at the EOL below.

Reply:
I tried with `\` but not with `|`. I will check that.

Reply:
perfect, working